govcd/sample_govcd_test_config.json

```diff
@@ -23,7 +23,7 @@
 			"description": "my cat for loading",
 			"catalogItemDescription": "my item to create vapps"
 		},
-		"network": "mynet",
+		"networks": ["mynet", "mynet2"],
```

Comment:
What's the required number of networks? I see that we have some tests that check for length of this field as being `>= 2`.
* Is two networks sufficient? 
* Do they need to be of different type?